policy_tests/Makefile

```diff
@@ -35,75 +35,136 @@ webapp_TIMEOUT = 360
 webapp: CONFIG=webapp
 webapp: all
 
-# config for bare-metal on QEMU
-bare-qemu_SIM = qemu
-bare-qemu_TESTS = bare
-bare-qemu_RUNTIME = bare
-bare-qemu_MODULE = osv
-bare-qemu_ARCH = rv32
-bare-qemu_GPOLICIES = 
-bare-qemu_POLICIES = cfi,heap,rwx,stack,taint,threeClass,none,testSimple
-bare-qemu_POLICIES = cfi,heap,rwx,stack,threeClass,none,testSimple
-bare-qemu_XDIST = -n $(JOBS) # run in parallel
-bare-qemu_TIMEOUT = 360
-bare-qemu: CONFIG=bare-qemu
-bare-qemu: all
-
-# config for 64-bit bare-metal on QEMU
-bare64-qemu_SIM = qemu
-bare64-qemu_TESTS = bare64
-bare64-qemu_RUNTIME = bare
-bare64-qemu_ARCH = rv64
-bare64-qemu_MODULE = osv
-bare64-qemu_GPOLICIES = 
-bare64-qemu_POLICIES = rwx,none,heap,taint,testSimple
-bare64-qemu_XDIST = -n $(JOBS) # run in parallel
-bare64-qemu_TIMEOUT = 360
-bare64-qemu: CONFIG=bare64-qemu
-bare64-qemu: all
-
-# config for bare-metal on VCU118 board
-bare-vcu118_SIM = vcu118
-bare-vcu118_TESTS = bare
-bare-vcu118_RUNTIME = bare
-bare-vcu118_MODULE = osv
-bare-vcu118_ARCH = rv32
-bare-vcu118_GPOLICIES = 
-bare-vcu118_POLICIES = heap,stack,rwx
-bare-vcu118_XDIST = -n $(JOBS) # run in parallel
-bare-vcu118_TIMEOUT = 360
-bare-vcu118_PYTEST_ARGS = --extra=+bitstream=$(BITSTREAM),+processor=P1
-bare-vcu118: CONFIG=bare-vcu118
-bare-vcu118: all
-
-# config for 64-bit bare-metal on VCU118 board
-bare64-vcu118_SIM = vcu118
-bare64-vcu118_TESTS = bare64
-bare64-vcu118_RUNTIME = bare
-bare64-vcu118_ARCH = rv64
-bare64-vcu118_MODULE = osv
-bare64-vcu118_GPOLICIES = 
-bare64-vcu118_POLICIES = heap,rwx
-bare64-vcu118_XDIST = -n $(JOBS) # run in parallel
-bare64-vcu118_TIMEOUT = 360
-bare64-vcu118_PYTEST_ARGS = --extra=+bitstream=$(BITSTREAM),+processor=P2
-bare64-vcu118: CONFIG=bare64-vcu118
-bare64-vcu118: all
+# config for 32-bit hifive bare-metal on QEMU
+bare-hifive32_SIM = qemu
+bare-hifive32_TESTS = bare
+bare-hifive32_RUNTIME = bare
+bare-hifive32_ARCH = rv32
+bare-hifive32_MODULE = osv
+bare-hifive32_GPOLICIES = 
+bare-hifive32_POLICIES = rwx,none,heap,taint,testSimple
```

Comment:
Is there a reason that stack and cfi have been removed from many of the sets of policies to run?

Reply:
Not a good one; I used copy and paste a lot when updating these targets and missed that the `POLICIES` variables were different for different sim/design combinations.